scripts/apps/instance-configuration/index.ts

```diff
@@ -0,0 +1,32 @@
+import {gettext} from 'core/utils';
+import {reactToAngular1} from 'superdesk-ui-framework';
+import {IBaseRestApiResponse, IUser} from 'superdesk-api';
+import {InstanceConfigurationSettings} from './components/InstanceConfiguration';
+
+export interface ISystemMessage extends IBaseRestApiResponse {
+    type: 'warning' | 'alert' | 'primary' | 'success';
+    is_active: boolean;
+    message_title: string;
+    message: string;
+    user_id: IUser['_id'];
+}
+
+angular.module('superdesk.apps.instance-configuration', [
+    'superdesk.core.activity',
+])
+    .component('sdInstanceConfigurationSettings', reactToAngular1(
+        InstanceConfigurationSettings,
+        [],
+        [],
+    ))
+
+    .config(['superdeskProvider', function config(superdesk) {
+        superdesk.activity('/settings/instance-configuration', {
+            label: gettext('Instance configuration'),
+            // tslint:disable-next-line: max-line-length
+            template: '<div class="sd-scrollable-inside-absolute-container"><sd-instance-configuration-settings></sd-instance-configuration-settings></div>',
+            category: superdesk.MENU_MAIN,
+            priority: 1000,
+            adminTools: true,
```

Comment:
do we show these to administrators only?

Reply:
No, these only describe position in UI. I have added `privileges: {users: 1},`, but it would probably be better to use another privilege, since this is about instance config, not users. Do you know what privilege could I use?

Reply:
we would have to add a new one, atm on the backend it checks if user is type administrator only